libs/datasets/timeseries.py

```diff
@@ -645,7 +646,21 @@ def print_stats(self, name: str):
             .groupby(common_fields.COMMON_FIELD_TO_GROUP, sort=False)
             .sum()
         )
-        print(f"Dataset {name}:\n{count}")
+        by_level = (
+            self.timeseries_bucketed_long.index.get_level_values(CommonFields.LOCATION_ID)
+            .map(dataset_utils.get_geo_data()[CommonFields.AGGREGATE_LEVEL])
+            .value_counts()
+            .rename(None)
```

Comment:
What does this `rename(None)` do?  Can't tell intuitively or from reading the `rename()` docs.  Probably deserves a comment if it's actually doing something.

Reply:
It clears the series name, which was confusing when the series was output using repr. But I changed to `to_string()` which doesn't output the name so removed the rename.